worlds/baylands.sdf

```diff
@@ -113,6 +113,7 @@
       <world_frame_orientation>ENU</world_frame_orientation>
       <latitude_deg>37.412173071650805</latitude_deg>
       <longitude_deg>-121.998878727967</longitude_deg>
+      <elevation>38</elevation>
```

Comment:
Is this going to be the elevation above the average mean sea level? If so, we need to handle this on the GPS plugin side since it reports both ellipsoidal altitude (WGS84) and average mean sea level (AMSL)

Reply:
@Jaeyoung-Lim Good question. I'm not sure exacty but I looked at another source that showed that the elevation of the park itself is between -1 and 1m (it is right by the sea side)
https://en-ca.topographic-map.com/map-bj51h/Sunnyvale/?center=37.41279%2C-121.99417&zoom=16&popup=37.41189%2C-121.99704
Perhaps it is therefore best left at 0?

Reply:
@frede791 It is actually irrelevant how close the water level is. The AMSL is defined through a geoid (e.g. EGM96: https://en.wikipedia.org/wiki/Earth_Gravitational_Model).

I recently had some issues with the GPS plugin that the reported altitude alignments were unrealistic (Currently the GPS plugin assumes that the ellipsoidal altitude and AMSL altitude are the same: https://github.com/PX4/PX4-Autopilot/blob/8a7573351182bcc7181bb0377e35f62b2ad267ff/src/modules/simulation/sensor_gps_sim/SensorGpsSim.cpp#L158-L159

So wanted to make sure we have are considering these since we are now taking into account geocoordinates.